packages/std/src/results/cosmos_msg.rs

```diff
@@ -84,6 +85,7 @@ pub enum CosmosMsg<T = Empty> {
     Wasm(WasmMsg),
     #[cfg(feature = "stargate")]
     Gov(GovMsg),
+    Eureka(EurekaMsg),
```

Comment:
We need to add a [capability](https://github.com/CosmWasm/cosmwasm/blob/main/docs/CAPABILITIES.md) for this and feature-gate this line, since Eureka will probably not be available yet on all chains when we release the implementation.

Reply:
Good catch!